apps/web/cypress/plugins/index.ts

```diff
@@ -138,5 +140,62 @@ module.exports = (on, config) => {
         templates,
       };
     },
+    async makeBlueprints() {
+      const dal = new DalService();
+      await dal.connect('mongodb://localhost:27017/novu-test');
+
+      const userService = new UserService();
+      const user = await userService.createUser();
+
+      const notificationGroupRepository = new NotificationGroupRepository();
+      const organizationService = new OrganizationService();
+      const environmentService = new EnvironmentService();
+
+      let organization = await organizationService.getOrganization(config.env.BLUEPRINT_CREATOR);
+      if (!organization) {
+        organization = await organizationService.createOrganization({ _id: config.env.BLUEPRINT_CREATOR });
+        await environmentService.createEnvironment(organization._id, 'Production');
+      }
+      const productionEnvironment = await environmentService.getProductionEnvironment(organization._id);
+
+      const generalGroup = await notificationGroupRepository.findOne({
+        name: 'General',
+        _environmentId: productionEnvironment._id,
+        _organizationId: organization._id,
+      });
+      if (!generalGroup) {
+        await notificationGroupRepository.create({
+          name: 'General',
+          _environmentId: productionEnvironment._id,
+          _organizationId: organization._id,
+        });
+      }
+
+      const notificationTemplateService = new NotificationTemplateService(
+        user._id,
+        organization._id,
+        productionEnvironment._id
+      );
+
+      const templatesCount = await notificationTemplateService.countTemplates();
+      if (templatesCount === 0) {
+        return await Promise.all([
+          notificationTemplateService.createTemplate({
+            noFeedId: true,
+            noLayoutId: true,
+            name: ':fa-solid fa-star: Super cool workflow',
+            isBlueprint: true,
+          }),
+          notificationTemplateService.createTemplate({
+            noFeedId: true,
+            noLayoutId: true,
+            name: ':fa-solid fa-lock: Password reset',
+            isBlueprint: true,
+          }),
+        ]);
```

Comment:
In the current PR state, this part could cause flakiness where we create the second one before the first one, and the test could fail.
It will be resolved on the next commit to this branch. 

p.s the solution will be a side effect 😅
p.s.s the solution is that we will add here _id and it will be in the same order the API will have, therefore the list order will be always the same.